blueprints/node-webkit/files/nw-runner.js

```diff
@@ -0,0 +1,112 @@
+var exec = require('child_process').exec;
+var NW_VERSION = '0.12.0';
+var os = require('os');
+var fs = require('fs');
+
+runNw('./node_modules/nw/bin/nw');
```

Comment:
For the NW.js binary, it might make more sense to re-use `ember-cli-node-webkit/lib/helpers/find-nw.js` since that takes care of extracting the correct NW path, as well as dealing with cases where that path may be customized by users (via the `NW_PATH` environment variable).